train_management/templates/train_management/dashboard.html

```diff
@@ -152,11 +152,22 @@
             <div class="card-text">
               <ul>
                 <li>
-                  Erstellen und bearbeiten von <a href="{% url 'day-planning-list' %}">Tagesplanungen.</a>
+                  Personal:
                   <ul>
-                    <li>Züge können mit Formation hinzugefügt und bearbeitet werden.</li>
-                    <li>Funktionen können hinzugefügt und bearbeitet werden.</li>
-                    <li>Das Briefing kann erstellt werden.</li>
+                    <li>Personal kann Gruppen zugeordnet werden. Die Gruppen geben im System Berechtigungen.</li>
```

Comment:
Gibt es hier absichtlich keine Übersetzung?

Reply:
these texts shall be removed again soon. no need to translate them